osu.Game/Configuration/SessionStatics.cs

```diff
@@ -20,6 +20,12 @@ protected override void InitialiseDefaults()
             SetDefault(Static.LastHoverSoundPlaybackTime, (double?)null);
             SetDefault<APISeasonalBackgrounds>(Static.SeasonalBackgrounds, null);
         }
+
+        public void ResetValues()
+        {
+            ConfigStore.Clear();
```

Comment:
Actually I think this will break, since any existing usages of the old bindables will be lost.

Going to have to re-think this.

Reply:
ideally i'd use `ConfigManager.getDefault()`, but as mentioned previously, that method is private.

is there a reason in particular it's private or can it be set to public?

Reply:
The best way is actually calling `.SetDefault()` on the bindables, but this requires having the non-interface representation of them, which I don't think is accessible.

Reply:
I've reverted back to how it was before for now, so let me know if you think of a better way to do this.